<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Win_Fight</class>
 <widget class="QWidget" name="Win_Fight">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1708</width>
    <height>1033</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>130</number>
   </property>
   <property name="topMargin">
    <number>70</number>
   </property>
   <property name="rightMargin">
    <number>130</number>
   </property>
   <property name="bottomMargin">
    <number>70</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_5" stretch="1,0">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>80</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,0">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,0">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="layout_monster" stretch="1,0">
           <property name="spacing">
            <number>25</number>
           </property>
           <property name="leftMargin">
            <number>30</number>
           </property>
           <item>
            <widget class="QLabel" name="img_monster">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>200</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>200</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel{
	border-radius:100px;
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="layout_monster2" stretch="1,1,1,2">
             <property name="spacing">
              <number>15</number>
             </property>
             <item>
              <spacer name="verticalSpacer_5">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="W_StatusBar" name="monsterStatus" native="true">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>30</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="W_AnimatedProgressBar" name="monsterLife" native="true">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>25</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>25</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_4">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>50</number>
           </property>
           <property name="topMargin">
            <number>30</number>
           </property>
           <property name="rightMargin">
            <number>50</number>
           </property>
           <property name="bottomMargin">
            <number>30</number>
           </property>
           <item>
            <widget class="MonsterFightView" name="fight_view" native="true">
             <property name="minimumSize">
              <size>
               <width>800</width>
               <height>400</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>800</width>
               <height>400</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,0,0,0">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>20</number>
         </property>
         <property name="bottomMargin">
          <number>20</number>
         </property>
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>20</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <property name="spacing">
              <number>20</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <property name="spacing">
                <number>0</number>
               </property>
               <item row="1" column="0">
                <widget class="QPushButton" name="button_useSpell">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>140</width>
                   <height>140</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>140</width>
                   <height>140</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QPushButton{
	border:1px solid black;
	border-image:url(:/icons/button/buttonFight_spell.png);
}

QPushButton::hover{
	border-image:url(:/icons/button/buttonFight_spell_h.png);
}

QPushButton::pressed{
	border-image:url(:/icons/button/buttonFight_spell_p.png);
}

QPushButton:disabled{
	border-image:url(:/icons/button/buttonFight_spell_d.png);
}
/*QPushButton{
	border:1px solid black;
	border-image:url(:/icons/button/buttonFight_4.png);
}

QPushButton::hover{
	border-image:url(:/icons/button/buttonFight_4_hover.png);
}

QPushButton::pressed{
	border-image:url(:/icons/button/buttonFight_4_pressed.png);
}

QPushButton::disabled{
	border-image:url(:/icons/button/buttonFight_4_disabled.png);
}*/</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="1" column="2">
                <widget class="QPushButton" name="button_lightAttack">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>140</width>
                   <height>140</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>140</width>
                   <height>140</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QPushButton{
	border:1px solid black;
	border-image:url(:/icons/button/buttonFight_2.png);
}

QPushButton::hover{
	border-image:url(:/icons/button/buttonFight_2_hover.png);
}

QPushButton::pressed{
	border-image:url(:/icons/button/buttonFight_2_pressed.png);
}

QPushButton::disabled{
	border-image:url(:/icons/button/buttonFight_2_disabled.png);
}</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QPushButton" name="button_heavyAttack">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>140</width>
                   <height>140</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>140</width>
                   <height>140</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QPushButton{
	border:1px solid black;
	border-image:url(:/icons/button/buttonFight_1.png);
}

QPushButton::hover{
	border-image:url(:/icons/button/buttonFight_1_hover.png);
}

QPushButton::pressed{
	border-image:url(:/icons/button/buttonFight_1_pressed.png);
}

QPushButton::disabled{
	border-image:url(:/icons/button/buttonFight_1_disabled.png);
}</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QPushButton" name="button_flee">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>140</width>
                   <height>140</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>140</width>
                   <height>140</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QPushButton{
	border:1px solid black;
	border-image:url(:/icons/button/buttonFight_3.png);
}

QPushButton::hover{
	border-image:url(:/icons/button/buttonFight_3_hover.png);
}

QPushButton::pressed{
	border-image:url(:/icons/button/buttonFight_3_pressed.png);
}

QPushButton::disabled{
	border-image:url(:/icons/button/buttonFight_3_disabled.png);
}</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>30</number>
             </property>
             <property name="topMargin">
              <number>20</number>
             </property>
             <property name="rightMargin">
              <number>20</number>
             </property>
             <property name="bottomMargin">
              <number>20</number>
             </property>
             <item>
              <widget class="QProgressBar" name="data_playerStamina">
               <property name="minimumSize">
                <size>
                 <width>10</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>10</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QProgressBar {
	border:2px solid #575757;
	border-radius:2px;
	background-color: qlineargradient(x1: 0, y1: 0, x2: 1, y2: 1, stop: 0 #393939, stop: 1 #393939);
}

QProgressBar::chunk {
	background-color: qlineargradient(x1: 0, y1: 1, x2: 1, y2: 1,  stop: 1 #ffd800, stop: 0 #ffd800);
	border-radius:2px;
}</string>
               </property>
               <property name="value">
                <number>24</number>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="textVisible">
                <bool>false</bool>
               </property>
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_7">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,3">
           <property name="spacing">
            <number>0</number>
           </property>
           <item alignment="Qt::AlignRight|Qt::AlignVCenter">
            <widget class="QLabel" name="img_player">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>150</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>150</width>
               <height>150</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel{
	border-radius:75px;
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,1,1,2">
             <property name="spacing">
              <number>8</number>
             </property>
             <property name="leftMargin">
              <number>12</number>
             </property>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="W_AnimatedProgressBar" name="heroLife" native="true">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>25</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>25</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="W_AnimatedProgressBar" name="heroMana" native="true">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>12</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>12</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="W_StatusBar" name="heroStatus" native="true">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>30</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="3">
       <property name="spacing">
        <number>30</number>
       </property>
       <property name="leftMargin">
        <number>20</number>
       </property>
       <property name="rightMargin">
        <number>20</number>
       </property>
       <item>
        <widget class="QGraphicsView" name="itemView">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>125</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>125</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QGraphicsView{
		background-color:rgba(250,250,250,5);
    	border: none;
    	margin: 0px 0px 0px 0px;
}
QGraphicsView QScrollBar::horizontal {
        background: none;
        height: 12px;
		border-radius:2px;
}
QGraphicsView QScrollBar::groove {
        background: none;
}
QGraphicsView QScrollBar::left-arrow{
	background: none;
}
QGraphicsView QScrollBar::right-arrow{
	background: none;
}
QGraphicsView QScrollBar::sub-line:horizontal{
	background: none;
}
QGraphicsView QScrollBar::add-line:horizontal{
	background: none;
}
QScrollBar::sub-page:horizontal {
	background: none;
}
QGraphicsView QScrollBar::handle {
        background: rgba(250,250,250,15);
		border-radius:5px;
		max-width:10px;
}
/*
QGraphicsView QScrollBar::add-line:horizontal {
       background: white;
        width: 0px;
        subcontrol-position: left;
        subcontrol-origin: margin;
}
QGraphicsView QScrollBar::sub-line:horizontal {
        background: white;
        width: 0 px;
        subcontrol-position: right;
       subcontrol-origin: margin;
}*/</string>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOn</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MonsterFightView</class>
   <extends>QWidget</extends>
   <header>monsterfightview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>W_AnimatedProgressBar</class>
   <extends>QWidget</extends>
   <header>win_fight.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>W_StatusBar</class>
   <extends>QWidget</extends>
   <header>win_fight.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
